improver/utilities/cube_extraction.py

```diff
@@ -130,6 +115,8 @@ def parse_constraint_list(
     Returns:
         - A combination of all the constraints that were supplied.
         - A dictionary of unit keys and values
+        - A list of lists containing the min and max values for a longitude constraint
```

Comment:
You're right - in one of my original attempts I allowed for multiple uses of `--constraint "longitude=..."`, I later removed it because I couldn't think of a use case to constrain the same coordinate multiple time. I'll fix the docstring now.